src/Stripe.net/Entities/StripeEphemeralkeyAssociatedObject.cs

```diff
@@ -0,0 +1,10 @@
+using Newtonsoft.Json;
+
+namespace Stripe
+{
+    public class StripeEphemeralkeyAssociatedObject : StripeEntityWithId
```

Comment:
I think the casing here should be the same as the other entity class (i.e. "EphemeralKey" instead of "Ephemeralkey").

Reply:
ah nice catch done.